### ISSUE TITLE
[Addon Confliction] Venture Plan dark text with NDui

### ISSUE THREAD

**Issue by JourneyOver** [siweia/NDui#132]
[Venture Plan](https://www.townlong-yak.com/addons/venture-plan) has recently released for the mission table, but there is an issue with it when using NDui. The text on the screen is so dark and blurry that it is hard to read.

What it looks like with NDui:
![](https://user-images.githubusercontent.com/13910285/103111103-01961980-460f-11eb-9d28-41a520aa75ab.png)

more than likely will need something done like what you have done with War Plan

**Comment by JourneyOver** [comment]
@siweia Works great, but there does seem to be an issue every now and then where it seems to default back to the black/blurry text. Not sure what causes it as it doesn't happen often and when it does a /reload always seems to fix it.

**Comment by siweia** [comment]
I just had a glimpse that day, seems useless atm.
Might have a go for the coming week.

**Comment by JourneyOver** [comment]
@siweia if your calling the addon useless, I don't completely find it useless myself nor do a lot of other people that it's been mentioned too, hell it even got spotlighted on wowhead recently xD 

Like yea I don't care much for the layout change that the addon does to the adventure table, but it adds a thing called the `Cursed Adventurer's Guide` in the actual adventure screen, that upon clicking will show if the adventure will succeed or not. You still have to add the companions and troops manually but unlike other addons (and several weak auras) that have been created/updated for the table so far, this one takes into account the abilities (and positions as well from the looks of things).

I've only ran into 2 weak auras that take into account a small number of abilities, and have yet to find a different addon that takes into account any abilities other than this one so far. 

With an easy way to send in the adventure logs as well for missions (after the mission succeeds or fails) that don't have a prediction on win/lose/being wrong or abilities that haven't been accounted for, and he seems to be actively working on it a lot more than the devs of the other addons/weakauras.

**Comment by siweia** [comment]
I knew its adventure calculation is much better than others, but the addon for now is not as good as its former, MasterPlan and WarPlan.
Hopefully the author would bring in more features in the future.

**Comment by JourneyOver** [comment]
Well it has only been released for about a week (released on the 18th), but I'm sure that over time it will grow to be really great just like master plan and war plan. Unlike old expansion mission tables which was a lot easier to figure out, blizzard had made the mission table for shadowlands so much more annoying to deal with when it comes to being able to simulate battles and all that due to the abilities and positions and all that shit having to be accounted for >.< but so far the simulation that venture plan has in so far for the adventurer's guide has been pretty great (though I'm a night fae so I can pretty much cheese any battle easily anyways lmfao)

**Comment by JourneyOver** [comment]
@siweia Looks like an update happened to venture plan within the past day and the black text is back again along with this error message:

```
...\NDui\Modules\Skins\Blizzard\Blizzard_GarrisonUI.lua:1005: attempt to call method 'SetTextColor' (a nil value)
Count: 1

Call Stack:
[string "@AddOns\TradeSkillMaster\LibTSM\Service\ErrorHandler.lua"]:758: in function <...Ons\TradeSkillMaster\LibTSM\Service\ErrorHandler.lua:717>
[string "=[C]"]: in function `SetTextColor'
[string "@AddOns\NDui\Modules\Skins\Blizzard\Blizzard_GarrisonUI.lua"]:1005: in function <...\NDui\Modules\Skins\Blizzard\Blizzard_GarrisonUI.lua:1002>
[string "@AddOns\NDui\Modules\Skins\Blizzard\Blizzard_GarrisonUI.lua"]:1068: in function <...\NDui\Modules\Skins\Blizzard\Blizzard_GarrisonUI.lua:1062>
[string "@AddOns\NDui\Modules\Skins\Blizzard\Blizzard_GarrisonUI.lua"]:1094: in function <...\NDui\Modules\Skins\Blizzard\Blizzard_GarrisonUI.lua:1083>
```

**Comment by siweia** [comment]
Should be fixed in 6.6.9
